man/systemd-firstboot.xml

```diff
@@ -34,12 +34,22 @@
   <refsect1>
     <title>Description</title>
 
-    <para><command>systemd-firstboot</command> initializes basic system settings interactively during the
-    first boot, or non-interactively on an offline system image. The service is started during boot if
-    <varname>ConditionFirstBoot=yes</varname> is met, which essentially means that <filename>/etc/</filename>
-    is unpopulated, see
+    <para>The <command>systemd-firstboot.service</command> is one of the units which are used to initialize
+    the machine during "First Boot", i.e. when the system is freshly installed or after a factory reset. The
```

Comment:
again, this still doesn't say that "systemd-firstboot" is about writing out *configuration*. it just says "initialize the machine", but that could be understood as runtime state init, which this is not about. See my earlier comments.

Reply:
not addressed? what  does "initialize the machine"? to me that sounds more like runtime init, not like populating /etc

Reply:
It is addressed in the later part of the paragraph. This first sentence provides an introduction, but who does what and other details are described right after.